docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -3,55 +3,52 @@
 :source-highlighter: highlightjs
 
 **{ProductName}** +
-A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.
+
+**build pipeline** +
+A Tekton PipelineRun which converts from source code to a built and tested artifact in a container registry.
 
 **build pipeline customization** +
-The ability to update and manage build pipelines for each component in an application. 
+The ability to update and manage build pipelines for each component in an application.
+
+**build-time tests** +
+A specific TaskRun which is included in a build pipeline but does not required to convert from the source control to the produced artifact. This may include any number of functional or security checks produced on the source code or the produced artifact.
 
 **cluster** +
 A Kubernetes deployment with nodes that run containerized applications and a control plane that manages the nodes.
 
 **component** +
-An internal representation of a container image that {ProductName} builds and tests, using source code from a git repository. 
+An internal representation of a software artifact that {ProductName} builds and tests, using source code from a git repository. Components are stored an link:https://github.com/opencontainers/distribution-spec[OCI container registry] such as quay.io after they are built.
 
 **Enterprise Contract (EC)** +
 A set of release policies that you apply to your application snapshot. You can use the EC to prevent releases that are not compliant with Supply-chain Levels for Software Artifacts (SLSA) guidelines. 
 
 **integration test** +
-A pipeline that you set up in GitHub. When you add components, it tests each one individually, and then tests the application as a whole.
+A pipeline defined in source control that is run after the completion of a component's build pipeline. Integration tests are run against all components in a snapshot as a whole.
 
 **IntegrationTestScenario (ITS)** +
-A Kubernetes resource that defines a pipeline to run. The integration service uses the ITS to test application components.
+A Kubernetes resource that defines a pipeline to run for an integration test. The integration service uses the ITS to test application components.
```

Comment:
Technically the ITS just defines metadata about the test and contains a resolver for a tekton pipeline.  We should make that clear here so that users understand how their tests are run.

Reply:
Are you saying that you suggestion is making it clearer? The information that you commented feels like too much for the documentation.